broker/api.py

```diff
@@ -343,6 +346,9 @@ def update(
             queue = queue_all_cdn_update_tasks_for_operation
         else:
             queue = queue_all_alb_update_tasks_for_operation
+        if noop:
+            return UpdateServiceSpec(False)
```

Comment:
If `noop` is `True` then run this?  What does `UpdateServiceSpec` do?  I see you're explicitly passing in `False` compared to other similar calls here, so I'm assuming this is indicating that nothing should take place but a record of the action needs to still be made?

Reply:
`UpdateServiceSpec` is the class the osbapi package uses to represent the return value of an update call. The parameter controls whether or not the return indicates the update is async. We have no work to do, so this is ~not async~ _synchronous_.
The upshot is that `UpdateServiceSpec(False)` will return a 200, and `UpdateServiceSpec(True)` will return a 202

Reply:
Excellent, thanks!